testsuite/features/build_validation/smoke_tests/smoke_tests.template

```diff
@@ -70,6 +71,7 @@ Feature: Smoke tests for <client>
     And I reboot the "<client>" if it is a transactional system
 
 @skip_for_sle_micro_ssh_minion
+@skip_for_sl_micro
```

Comment:
We should not be skipping them, this is a supported feature for SL/SLE Micro.
Instead, we need to find a way to cover this with a package, if none available now.

Hopefully this PR will help on this https://github.com/uyuni-project/sumaform/pull/1702